app/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -146,7 +146,6 @@ public void setTileRenderThrowableListener( TileRenderThrowableListener tileRend
 
   public void requestRender() {
     mRenderIsCancelled = false;
-    mRenderIsSuppressed = false;
```

Comment:
So this call is removed here because we should control the resume of tile rendering, by calling `resumeRender`.